modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,534 @@
+Data directory filesystem type                     xfs           xfs              Warning   true
+Data directory is writable                         true          true             Fatal     true
+Data partition free space [GB]                     >= 10         1755.29          Warning   true
+Dir '/var/lib/redpanda/data' IRQs affinity set     true          true             Warning   true
+Dir '/var/lib/redpanda/data' IRQs affinity static  true          true             Warning   true
+Dir '/var/lib/redpanda/data' nomerges tuned        true          true             Warning   true
+Dir '/var/lib/redpanda/data' scheduler tuned       true          true             Warning   true
+Free memory per CPU [MB]                           2048 per CPU  7659             Warning   true
+Fstrim systemd service and timer active            true          true             Warning   true
+I/O config file present                            true          true             Warning   true
+Kernel Version                                     3.19          5.15.0-1056-aws  Warning   true
+Max AIO Events                                     >= 1048576    1048576          Warning   true
+Max syn backlog size                               >= 4096       4096             Warning   true
+NIC IRQs affinity static                           true          true             Warning   true
+NTP Synced                                         true          true             Warning   true
+RFS Table entries                                  >= 32768      32768            Warning   true
+Swap enabled                                       true          true             Warning   true
+Swappiness                                         1             1                Warning   true
+Transparent huge pages active                      true          true             Warning   true
+```
+
+[[redpanda-lmr]]
+=== Redpanda's latest version
+
+Check that Redpanda is running on the xref:https://github.com/redpanda-data/redpanda/releases[latest maintenance release^] on every node.
+
+Example:
+```bash
+/usr/bin/redpanda --version
+```
+
+Output:
+```bash
+v23.3.9 - f0a24fa4b60d55bd075c291e213807b5dfa74c0a
+```
+
+[[cpu-memory]]
+=== Correct CPUs and Memory configured
+
+Ensure that the correct number of CPUs and memory are being used by Redpanda.
+
+Example:
+
+```bash
+journalctl -u redpanda | grep "System resources"
+```
+
+Output:
+
+```bash
+Mar 25 12:16:18 ip-172-31-10-199 rpk[3957]: INFO  2024-03-25 12:16:18,105 [shard 0:main] main - application.cc:350 - System resources: { cpus: 8, available memory: 55.578GiB, reserved memory: 3.890GiB}
+```
+
+[[mounted-disks]]
+=== Disks are correctly mounted
+
+Ensure that the correct disks are mounted and if multiple devices are used, they are configured as RAID-0. Other RAID configurations can have significantly worse latencies. File system should be of type XFS. If XFS is unavailable, ext4 is an appropriate alternative. 
```

Comment:
Need help with the examples in this section. It's not clear to me what's the input/output

Reply:
This is broken up weird. In the original doc, we're really talking about two different validation paths.

The first one assume you're not using RAID and have a single device configured with an xfs file system installed on it.

The second one assumes you **are** using raid, so you need to do more ... but looking further, I think we need to treat these as two different operations.

First this should probably be a step to validate the file system type (which we can verify with the `df`). Then we should beak out the mdadm stuff to another to validate the raid type. Everyone will need to do the xfs/ext4 check. not everyone will have raid so they won't need to do the mdadm check.

Reply:
Confusing for sure. What, exactly, is the user looking for in these outputs? I don't think it is sufficient to say "Example" for each code block here. We need to be clear about what each example is doing, and what users are looking for. If their output is different, then fails?